src/sidebar/services/test/api-test.js

```diff
@@ -89,7 +45,7 @@ describe('sidebar.services.api', function () {
     angular.mock.module('h', {
       apiRoutes: fakeApiRoutes,
       auth: fakeAuth,
-      settings: {apiUrl: 'http://example.com/api/'},
+      settings: {apiUrl: 'https://example.com/api/'},
```

Comment:
Can you help me understand the meaning/impact of changing protocol here to `https`? Does it have relevancy to the tests?

Reply:
The only reason I changed this was to make it a bit easier to update the route definitions in future by simplifying the processing that needs to be done on the JSON data fetched from https://hypothes.is/api/. The reason for not just using the original JSON data directly is to make sure the client is actually using the full URLs from the JSON. I'm starting to doubt that is justified TBH.

Reply:
Can you help me understand the downside of having `hypothes.is` hard-coded here? This is all fake, right? For tests?